pkg/api/api.proto

```diff
@@ -132,7 +132,42 @@ message CreateReleaseRequest {
   string displayVersion = 10;
 }
 
+message CreateReleaseResponseSuccess {
+}
```

Comment:
overall fine, but actual "diff" is missing still

Reply:
success never has a diff (because no version existed previously in that case).